compiler/circle-partitioner/src/PartitionRead.cpp

```diff
@@ -69,13 +92,33 @@ luci::PartitionTable parse_table(const crew::Sections &sections)
       for (auto &item : items)
       {
         if (item.first == _key_underscore)
-          table.default_group = item.second;
+        {
+          if (table.comply == luci::PartitionTable::COMPLY::OPCODE)
+            table.default_group = item.second;
+        }
         else
         {
           table.byopcodes.emplace(item.first, item.second);
         }
       }
     }
+    else if (section.name == _section_OPNAME)
+    {
+      auto &items = section.items;
+
+      for (auto &item : items)
+      {
+        if (item.first == _key_underscore)
+        {
+          if (table.comply == luci::PartitionTable::COMPLY::OPNAME)
+            table.default_group = item.second;
+        }
+        else
+        {
+          table.byopnames.emplace(item.first, item.second);
+        }
+      }
+    }
```

Comment:
(optional) Just for future revision, this is what I think it could be considered.
1. As I said in another PR, if `Sections` was map, it would be easy to implement these kind of logic.
2. I think there is only one section for each comply. So, like `find_section` function would help. Then, duplicate codes can be removed.
```cxx
..
// read other sections
section = find_section(sections, table.comply) // or, sections[table.comply] if map.
auto &items = section.items;
..
```
Or, just inserting `break` at each end of `if` condition in `for` loop would help.

Reply:
Hope to see the draft if you have some time to prepare :)